docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2114 @@
+|Argument |Type |Description
+
+|`column` ***
+
+Aliases: `field`, `c`
+|`string`
+|The column or field to attach the filter to
+
+|`value` ***
+
+Aliases: `v`, `val`
+|`string`
+|The value to match exactly, including white space and 
+capitalization
+
+|`filterGroup`
+|`string`
+|Group name for the filter
+|===
+
+*Returns:* `filter`
+
+
+[float]
+[[f_fns]]
+== F
+[[filterrows_fn]]
+=== `filterrows`
+
+Filters rows in a `datatable` based on the return value of a subexpression.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Aliases: `fn`, `exp`, `expression`
+|`boolean`
+|An expression to pass into each row in the `datatable.`
+The expression should return a `boolean`. A `true` value preserves the row, 
+and a `false` value removes it.
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[filters_fn]]
+=== `filters`
+
+Collects element filters on the workpad, usually to provide them to a data source.
+
+*Accepts:* `null`
+
+*Returns:* `filter`
+
+
+[float]
+[[font_fn]]
+=== `font`
+
+Creates font style.
+
+*Accepts:* `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`size`
+|`number`
+|The font size in pixels 
+
+Default: `14`
+
+|`lHeight` 
+
+Alias: `lineHeight`
+|`number`, `null`
+|The line height in pixels
+
+|`family`
+|`string`
+|An acceptable CSS web font string 
+
+Default: `"'Open Sans', Helvetica, Arial, sans-serif"`
+
+|`color`
+|`string`
+|The text color
+
+|`weight`
+|`string`
+|The font weight, for example, `"normal"`, `"bold"`, `"bolder"`, `"lighter"`, 
+`"100"`, `"200"`, `"300"`, `"400"`, `"500"`, `"600"`, `"700"`, `"800"`, `"900"` 
+
+Default: `"normal"`
+
+|`underline`
+|`boolean`
+|Underline the text? 
+
+Default: `false`
+
+|`italic`
+|`boolean`
+|Italicize the text? 
+
+Default: `false`
+
+|`align`
+|`string`
+|The horizontal alignment of text 
+
+Default: `left`
+|===
+
+*Returns:* `style`
+
+
+[float]
+[[formatdate_fn]]
+=== `formatdate`
+
+Format a valid date string or number of milliseconds since epoch using https://momentjs.com/[MomentJS]
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|MomentJS Format with which to bucket, see the https://momentjs.com/docs/#/displaying/[MomentJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[formatnumber_fn]]
+=== `formatnumber`
+
+Turns a `number` into a `string` using a NumeralJS format.
+
+*Accepts:* `number`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|NumeralJS format string, see the http://numeraljs.com/#format[NumeralJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[g_fns]]
+== G
+[[getCell_fn]]
+=== `getCell`
+
+Fetches a single cell in a table.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Aliases: `column`, `c`
+|`string`
+|The name of the column value to fetch, defaults to the first column
```

Comment:
This should be two sentences:

`The name of the column value to fetch. Defaults to the first column.`

Reply:
How about `The name of the column to fetch the value from. If not provided, the value will be retrieved from the first column.`?

Reply:
The first part is perfect, but for the second part, could we change it to `the value is retrieved from the first column.`